composer.json

```diff
@@ -13,7 +13,8 @@
     "php": ">=7.4",
     "yahnis-elsts/plugin-update-checker": "5.1",
     "ua-parser/uap-php": "dev-master",
-    "aws/aws-sdk-php": "^3.300"
+    "aws/aws-sdk-php": "^3.300",
+    "woocommerce/action-scheduler": "dev-trunk"
```

Comment:
We should lock this to a specific version, otherwise this will end up potentially pulling in new changes anytime we build a release, which could cause unexpected problems